server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -279,6 +279,13 @@ long getThrottleTimeInMillis() {
         boolean isThrottled() {
             return lock != NOOP_LOCK;
         }
+
```

Comment:
I would like to add a comment that this is to be used in assertions and tests only.